### PR TITLE
Update UConn-OSG.yaml

### PR DESCRIPTION
**Pull request by rjones30**
Update the resource topology for the UConn-OSG site.

### CODE REVIEW

File: topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml [hunk 1]
```diff
@@ -112,4 +106,38 @@ Resources:
       StorageCapacityMax: 50
       StorageCapacityMin: 25
       TapeCapacity: 0
+      UConn-OSG_CE:
```
Comment (edquist) [inline topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml]:
```suggestion
  UConn-OSG_CE:
```

Looks like this is starting a new resource, so I'm guessing the indent should be reset..?

File: topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml [hunk 2]
```diff
@@ -67,7 +71,7 @@ Resources:
           uri_override: gluskap.phys.uconn.edu:2811
     VOOwnership:
       Gluex: 100
-  UConn-OSG_SE:
+  GlueX_submit_host:
```
Comment (edquist) [inline topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml]: Just a note for `GlueX_submit_host` - resource names need to be globally unique, so if there might other GlueX submit nodes outside of UConn-OSG, you may want to consider including UConn-OSG in the name somehow.

File: topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml (hunk 1 above)
Comment (rjones30) [reply topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml]: Please fix it, I cannot see it in full context here but it looks like there are too many spaces in front of this resource name. -rtj

File: topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml (hunk 2 above)
Comment (rjones30) [reply topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml]: Ok, I thought the resource namespace is scoped by the site and resource group names. Wasn't that implied by the slash construction in the full resource path? Can I retract this pull request and try again?

Comment (edquist) [reply topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml]: Unfortunately both the resource group and resource group names must be unique across the entire OSG topology, despite them appearing at different parts of the hierarchy.

Yes feel free to make a new PR.

**Comment by rjones30** [comment]
How can I get back my submitted fixes so I can revise them? If I close the
pull request, won't they be deleted?
-Richard

**Comment by edquist** [comment]
The file as you submitted it was:
https://github.com/opensciencegrid/topology/blob/a4fbf4247b2c0d3d99d1bf9f2207f5a7fbc5f277/topology/University%20of%20Connecticut/UConn-OSG/UConn-OSG.yaml

Which consisted of this changeset:
https://patch-diff.githubusercontent.com/raw/opensciencegrid/topology/pull/526.patch

If you close the PR, these will all still be visible, both in this PR after it is closed, and also in your
[`patch-1` branch](/rjones30/OSG-topology/blob/patch-1/topology/University%20of%20Connecticut/UConn-OSG/UConn-OSG.yaml).  Closing the PR does not delete your changes from your branch.

If you are handy with git, you can do
```
$ git clone https://github.com/rjones30/OSG-topology
$ cd OSG-topology
$ git checkout patch-1
```
Then edit `topology/University of Connecticut/UConn-OSG/UConn-OSG.yaml`, then
```
$ git commit -a  # add your commit message
$ git push
```

**Pull request closed without merging**